cumulus_library/studies/core/core_templates/medicationrequest.sql.jinja

```diff
@@ -0,0 +1,59 @@
+{% import 'core_utils.jinja' as utils %}
+CREATE TABLE core__medicationrequest AS
+WITH temp_mr AS (
+    SELECT 
+        {{- utils.basic_cols(
+                'medicationrequest',
+                'mr',
+                ['id', 'status', 'intent']
+            ) 
+        }},
+        {{- utils.date_cols_from_str(
+                'medicationrequest',
+                'mr',
+                ['authoredon'],
+                schema
+            ) 
+        }},
+        {{- utils.truncate_date_cols(
+                'medicationrequest',
+                'mr',
+                [
+                    ('authoredon', 'month')
+                ],
+                schema
+            ) 
+        }},
+        {{- utils.nullable_cols(
+                'medicationrequest',
+                'mr',
+                [
+                    'display',
+                    ('subject', 'reference', 'subject_ref')
+                ],
+                schema
```

Comment:
nit; do you need to pass this down? Can the method just grab it from the global variables itself? Don't know if your brain would like or hate that, but might save some lines of code.

Reply:
I was testing a similar thing in the `quality` study and I think they cannot grab it from the global variables after all.  :-/